modules/con_enabling-config_management_plugins_in_an_argo_cd_cr.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-config_management_plugins_in_an_argo_cd_cr_{context}"]
+= Enabling Config Management Plugins in an Argo CD CR
+
+Argo CD provides support for Helm, Jsonnet, and Kustomize as built-in config management tools. To use a different config management tool, or to enable features not provided by the built-in config management tools, you can use the Config Management Plugin (CMP).
+
+In Argo CD, the Config Management Plugin (CMP) is specified as a sidecar container for the Argo CD repo server container. For more information, see _Config Management Plugins_.
```

Comment:
```suggestion
In Argo CD, the CMP is specified as a sidecar container for the Argo CD repo server container. For more information, see "Config Management Plugins".
```

- Acronym is already expanded above.
- See - https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

Reply:
Fixed.